docs/source/upgrade.rst

```diff
@@ -27,6 +27,14 @@ See the discussion in `#2982 <https://github.com/chainer/chainer/pull/2982>`_ fo
 
 This change does not break the existing code; you can safely continue to use updater classes directly under ``chainer.training`` but it is now encouraged to use ``chainer.training.updaters`` instead.
 
+Namespace Changes for Optimizer Hooks
+-------------------------------------
+
+:doc:`reference/optimizer_hooks` are moved from ``chainer.optimizer.*`` to ``chainer.optimizer_hooks.*``.
+
```

Comment:
It would be helpful to show an concrete example such as `chainer.optimizer.WeightDecay` -> `chainer.optimizer_hooks.WeightDecay`.

Reply:
Thanks, fixed.